flowmachine/flowmachine/features/subscriber/unique_active_subscribers.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from flowmachine.features.subscriber.active_subscribers import ActiveSubscribers
+
+
+class UniqueActiveSubscribers(ActiveSubscribers):
+    @property
+    def column_names(self) -> List[str]:
+        return ["subscriber"]
+
+    def _make_query(self):
+        list = super()._make_query()
```

Comment:
Composition over inheritance, by which I mean to say: this should be a query which takes the `ActiveSubscribers` query as an argument and does something with it.